### DISCUSSION TITLE
Plugin videos de YouTube

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#32] [Announcements]
# Plugin videos de YouTube

Inserta videos de YouTube agregando una lista de IDs en el campo de texto. Los videos no cargarán hasta que el usuario haga click

https://www.zkreations.com/2024/01/widget-videos-de-youtube-lazy-load-para.html

**Comment by TakoMics** [comment]
¡Santo cielo, Daniel! No tienes idea de cuánto he esperado el resurgimiento de este aporte en versión del formato de la nueva generación de tus plantillas.

En antaño, cuando creaba un blog, era una de las primeras cosas que hacía siquiera antes de enfocarme en el banner, logo y diseño.

Mil, mil gracias Daniel, 

Aquí, como adjunto, el mismo widget en su versión de antaño. Donde reemplacé el icono de play, por el logo de YouTube sacado de Font Awesome

![Captura del widget YouTube Lazy Load](https://i.postimg.cc/7Zmt0PYM/Captura.png)

Un saludo Daniel, gracias nuevamente. ¡Feliz fin de semana!

**Reply by danieIabel** [reply]
Debo de decir que, en particular, tanto Maily como Bison tendrán un problema en el título del widget, debido a que en esas plantillas hice algo raro con los mismos, pero en todas las demás y en sus respectivas actualizaciones eso está corregido. No más pido paciencia en esos casos. Por lo demás, todo funcionará adecuadamente. Muchas gracias por comentar amigo.

**Comment by SergioARC92** [comment]
Hmmm, me encanta la idea pero seria interesante tener como una especie de galeria donde esten varios videos de youtube con su flechita y todo para escoger, algo parecido a la cosa que tiene wix pero es un buen aporte, muchas gracias por compartir..
